readthedocs/gold/templates/gold/subscription_form.html

```diff
@@ -141,4 +141,5 @@ <h3>{{ subscription_title }}</h3>
 
     <em>{% trans "All information is submitted directly to Stripe." %}</em>
   </form>
+</div>
```

Comment:
I don't find where this div opens.

Actually, it's at the same level than the `<form>`, but the `</form>` is one more level to the right.

I got confused with this one.

Reply:
It is opening here:
https://github.com/rtfd/readthedocs.org/blob/ae991c4f131e2dd22fd4ce61deec381ae33f3aa4/readthedocs/gold/templates/gold/subscription_form.html#L36

Reply:
OK. So, I think we should fix the indentation also, right?

Reply:
Did you mean to indent the content under the `{%block edit_content %}` to right?